driver/apdu/pcsc.c

```diff
@@ -113,9 +114,9 @@ static int pcsc_open_hCard_iter(int index, const char *reader, void *userdata)
     DWORD dwActiveProtocol;
 
     id = 0;
-    if (getenv(INTERFACE_SELECT_ENV))
+    if (getenv(ENV_IFID) != NULL)
     {
-        id = atoi(getenv(INTERFACE_SELECT_ENV));
+        id = (int) strtol(getenv(ENV_IFID), NULL, 10);
```

Comment:
If you will cast to int later, why not use atoi directly?

Reply:
Clang-Tidy: 'atoi' used to convert a string to an integer value, but function will not report conversion errors; consider using 'strtol' instead